packages/hoppscotch-app/components/realtime/Socketio.vue

```diff
@@ -43,487 +46,387 @@
-          blank
-          svg="external-link"
-          reverse
-          class="mb-4"
-        />
-      </div>
-      <div
-        v-if="authType === 'Bearer'"
-        class="flex flex-1 border-b border-dividerLight"
-      >
-        <div class="w-2/3 border-r border-dividerLight">
-          <div class="flex flex-1 border-b border-dividerLight">
-            <SmartEnvInput v-model="bearerToken" placeholder="Token" />
           </div>
-        </div>
-        <div
-          class="sticky h-full p-4 overflow-auto bg-primary top-upperTertiaryStickyFold min-w-46 max-w-1/3 z-9"
-        >
-          <div class="p-2">
-            <div class="pb-2 text-secondaryLight">
-              {{ $t("helpers.authorization") }}
-            </div>
-            <SmartAnchor
-              class="link"
-              :label="`${$t('authorization.learn')} \xA0 →`"
+          <div
+            v-if="authType === 'None'"
+            class="flex flex-col items-center justify-center p-4 text-secondaryLight"
+          >
+            <img
+              :src="`/images/states/${$colorMode.value}/login.svg`"
+              loading="lazy"
+              class="inline-flex flex-col object-contain object-center w-16 h-16 my-4"
+              :alt="`${t('empty.authorization')}`"
+            />
+            <span class="pb-4 text-center">
+              {{ t("socketio.connection_not_authorized") }}
+            </span>
+            <ButtonSecondary
+              outline
+              :label="t('app.documentation')"
               to="https://docs.hoppscotch.io/features/authorization"
               blank
+              svg="external-link"
+              reverse
+              class="mb-4"
             />
           </div>
-        </div>
-      </div>
+          <div
+            v-if="authType === 'Bearer'"
+            class="flex flex-1 border-b border-dividerLight"
+          >
+            <div class="w-2/3 border-r border-dividerLight">
+              <div class="flex flex-1 border-b border-dividerLight">
+                <SmartEnvInput v-model="bearerToken" placeholder="Token" />
+              </div>
+            </div>
+            <div
+              class="sticky h-full p-4 overflow-auto bg-primary top-upperTertiaryStickyFold min-w-46 max-w-1/3 z-9"
+            >
+              <div class="p-2">
+                <div class="pb-2 text-secondaryLight">
+                  {{ t("helpers.authorization") }}
+                </div>
+                <SmartAnchor
+                  class="link"
+                  :label="`${t('authorization.learn')} \xA0 →`"
+                  to="https://docs.hoppscotch.io/features/authorization"
+                  blank
+                />
+              </div>
+            </div>
+          </div>
+        </SmartTab>
+      </SmartTabs>
     </template>
     <template #secondary>
       <RealtimeLog
-        :title="$t('socketio.log')"
+        :title="t('socketio.log')"
         :log="log"
```

Comment:
There is a problem here, when you recieve a message on a topic, you get it on this format:

```
[topic name] message
```

The problem is that Log Entries can be expanded and their contents are parsed and the content type (whether it is JSON or not) is detected. With the current style,
we have an issue that the topic part gets considered as the body and is included in the message detection. We need to figure out a way to fix this issue. This same issue
is present in MQTT and SSE as well, they need to be handled there as well. (Please reach out with your solution before you code it in though)